packages/shared/src/UI/components/ERC20TokenList/index.tsx

```diff
@@ -106,8 +106,7 @@ export const ERC20TokenList = memo<ERC20TokenListProps>((props) => {
 
     const getPlaceHolder = () => {
         if (erc20TokensDetailedLoading) return <Placeholder message="Loading token lists..." />
-        if (searchedTokenLoading) return <Placeholder message="Loading token..." />
-        // if (assetsLoading) return <Placeholder message="Loading token assets..." />
+        if (searchedTokenLoading || searchedTokenLoading) return <Placeholder message="Loading token..." />
```

Comment:
The same variable is in the condition statement.

Reply:
> 

Deleted.